CONTRIBUTING.md

```diff
@@ -0,0 +1,92 @@
+# How to contribute
+
+Operator Controller is an Apache 2.0 licensed project and accepts contributions via GitHub pull requests (PRs).
+
+This document outlines some conventions on commit message formatting, contact points for developers, and other resources
+to help Operator Controller contributors.
+
+## Communication
+
+- Email: [operator-framework-olm-dev](mailto:operator-framework-olm-dev@googlegroups.com)
+- Slack: [#olm-dev](https://kubernetes.slack.com/archives/C0181L6JYQ2)
+- Google Group: [olm-gg](https://groups.google.com/g/operator-framework-olm-dev)
+- Weekly in Person Working Group Meeting: [olm-wg](https://github.com/operator-framework/community#operator-lifecycle-manager-working-group) 
+
+## Getting started
+
+- Fork the repository on GitHub.
+- Clone onto your local development machine via `git clone https://github.com/{$GH-USERNAME}/operator-controller`.
+- Add the operator-framework/operator-controller as an upstream.
+  remote: `git remote add upstream https://github.com/operator-framework/operator-controller`.
+- Create a new branch from the default `main` branch and begin development.
```

Comment:
There is a more detailed getting started section in README.md, so we may want to consider moving this section there.

Reply:
Done.